lib/pbench/agent/tool_meister.py

```diff
@@ -1316,121 +1316,120 @@ def _send_directory(self, directory, uri, ctx):
         target_dir = directory.name
         parent_dir = directory.parent
         tar_file = parent_dir / f"{target_dir}.tar.xz"
-        o_file = parent_dir / f"{target_dir}.tar.out"
-        e_file = parent_dir / f"{target_dir}.tar.err"
+        tar_args = [
+            self.tar_path,
+            "--create",
+            "--xz",
+            "--force-local",
+            f"--file={tar_file}",
+            target_dir,
+        ]
         try:
             # Invoke tar directly for efficiency.
-            with o_file.open("w") as ofp, e_file.open("w") as efp:
-                cp = subprocess.run(
-                    [
-                        self.tar_path,
-                        "--create",
-                        "--xz",
-                        "--force-local",
-                        f"--file={tar_file}",
-                        target_dir,
-                    ],
-                    cwd=parent_dir,
-                    stdin=None,
-                    stdout=ofp,
-                    stderr=efp,
-                )
+            cp = subprocess.run(
+                tar_args,
+                cwd=parent_dir,
+                stdin=None,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.STDOUT,
+            )
```

Comment:
Perhaps not in this PR...but I can dream....  😿

Reply:
Well we still need a `try` block because we are running the `tar` command directly and if the `tar_path` is somehow got corrupt it would result in an exception.

Reply:
True, if `self.tar_path` were to go bad between the call to `find_executable()` and here, we would get an exception raised by `run()`.  However, I would contend that that is so unlikely that it's not worth the cost and complication of trying to catch it locally -- I think that that's tantamount to a catastrophic failure, and we should deal with that at the top-most level if at all.